src/dbnode/storage/shard_test.go

```diff
@@ -1932,16 +1888,18 @@ func TestShardAggregateTiles(t *testing.T) {
 			BlockStart:          opts.Start,
 			BlockSize:           targetBlockSize,
 			VolumeIndex:         1,
-			PlannedRecordsCount: 2,
+			PlannedRecordsCount: 3,
```

Comment:
Just wondering why this has increased?

Reply:
To cover some additional logics; will make this self-documenting by introducing named vars.